auth/build.gradle

```diff
@@ -31,7 +31,8 @@ android {
 dependencies {
     implementation project(path: ':core')
     implementation 'net.openid:appauth'
-
+    implementation 'com.datatheorem.android.trustkit:trustkit:1.0.1@aar'
```

Comment:
I think we can leave out the cert pinning library for now as we aren't pinning against anything.